src/main/config/config_eeprom.c

```diff
@@ -83,6 +83,157 @@ typedef struct {
     uint32_t word;
 } PG_PACKED packingTest_t;
 
+#ifdef EEPROM_IN_EXTERNAL_FLASH
+bool loadEEPROMFromExternalFlash(void)
+{
+    const flashPartition_t *flashPartition = flashPartitionFindByType(FLASH_PARTITION_TYPE_CONFIG);
+    const flashGeometry_t *flashGeometry = flashGetGeometry();
+
+    uint32_t flashStartAddress = flashPartition->startSector * flashGeometry->sectorSize;
+
+    uint32_t totalBytesRead = 0;
+    int bytesRead = 0;
+
+    bool success = false;
+
+    do {
+        bytesRead = flashReadBytes(flashStartAddress + totalBytesRead, &eepromData[totalBytesRead], EEPROM_SIZE - totalBytesRead);
+        if (bytesRead > 0) {
+            totalBytesRead += bytesRead;
+            success = (totalBytesRead == EEPROM_SIZE);
+        }
+    } while (!success && bytesRead > 0);
+
+    return success;
+}
+#endif
+
+#ifdef EEPROM_IN_SDCARD
```

Comment:
Can `EEPROM_IN_EXTERNAL_FLASH` and `EEPROM_IN_SDCARD` be combined? If not this should probably be an `#elsif`.

Reply:
done